packages/backend-modules/documents/lib/mdast/process.js

```diff
@@ -43,7 +43,7 @@ const processRepoImageUrlsInContent = async (content, fn) => {
     }
   })
 
-  return Promise.all(fns.map((fn) => fn()))
+  return Promise.each(fns, fn => fn())
```

Comment:
I also tried to limit the concurrency with:
`Promise.map(fns, fn => fn(), { concurrency: 5 }`
but the asset server was still crashing.

Reply:
@lucianak